bio.html

```diff
@@ -0,0 +1,105 @@
+                    <path d="M4.81 1.37A6.5 6.5 0 0 1 14.56 7a.5.5 0 1 1-1 0 5.5 5.5 0 0 0-8.25-4.765.5.5 0 0 1-.5-.865Zm-.89 1.257a.5.5 0 0 1 .04.706A5.478 5.478 0 0 0 2.56 7a.5.5 0 0 1-1 0c0-1.664.626-3.184 1.655-4.333a.5.5 0 0 1 .706-.04ZM1.915 8.02a.5.5 0 0 1 .346.616l-.779 2.767a.5.5 0 1 1-.962-.27l.778-2.767a.5.5 0 0 1 .617-.346Zm12.15.481a.5.5 0 0 1 .49.51c-.03 1.499-.161 3.025-.727 4.533l-.07.187a.5.5 0 0 1-.936-.351l.07-.187c.506-1.35.634-2.74.663-4.202a.5.5 0 0 1 .51-.49Z"/>
+                    </svg></a>
+                <a href = "experience.html"><svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="currentColor" class="svg-icon" viewBox="0 0 1024 1024" version="1.1">
+                    <path d="M902.604162 233.211483 734.909859 233.211483l0-55.456033c0-30.458709-25.15389-55.428404-55.898101-55.428404L343.617012 122.327046c-30.744211 0-55.892985 24.969695-55.892985 55.428404l0 55.456033-167.69942 0c-30.738072 0-55.892985 24.935925-55.892985 55.423287l0 554.363853c0 30.492478 25.15389 55.434543 55.892985 55.434543l782.578532 0c30.744211 0 55.898101-24.942065 55.898101-55.434543L958.50124 288.63477C958.502263 258.147408 933.348374 233.211483 902.604162 233.211483L902.604162 233.211483zM343.617012 177.883363c0.033769-0.033769 0.061398-0.083911 0.11154-0.128937l335.193155 0c0.033769 0.045025 0.089028 0.072655 0.11768 0.128937l0 55.32812L343.650781 233.211483l0-55.32812L343.617012 177.883363zM902.604162 454.958866l-111.796202 0 0 138.599661c0 15.220656-12.577456 27.709085-27.949562 27.709085l-55.898101 0c-15.377222 0-27.949562-12.487406-27.949562-27.709085L679.010735 454.958866 343.617012 454.958866l0 138.599661c0 15.220656-12.571317 27.709085-27.949562 27.709085L259.770372 621.267613c-15.365966 0-27.943422-12.487406-27.943422-27.709085L231.826949 454.958866 120.024607 454.958866l0-55.4458 782.578532 0L902.603139 454.958866 902.604162 454.958866zM902.604162 454.958866"  />   
+                </svg></a>
+                <a href = "portfolio.html"><svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="currentColor" class = "protIcon" class="bi bi-card-image" viewBox="0 0 16 16">
+                    <path d="M6.002 5.5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z"/>
+                    <path d="M1.5 2A1.5 1.5 0 0 0 0 3.5v9A1.5 1.5 0 0 0 1.5 14h13a1.5 1.5 0 0 0 1.5-1.5v-9A1.5 1.5 0 0 0 14.5 2h-13zm13 1a.5.5 0 0 1 .5.5v6l-3.775-1.947a.5.5 0 0 0-.577.093l-3.71 3.71-2.66-1.772a.5.5 0 0 0-.63.062L1.002 12v.54A.505.505 0 0 1 1 12.5v-9a.5.5 0 0 1 .5-.5h13z"/>
+                </svg></a>
+            </div>
+        </div>
+    </header>
+    <main>
+        <br>
+        <br>
+        <div id = "mainParent">
```

Comment:
Instead of "main parent" maybe this would be a good opportunity for a semantic HTML tag?